llvm/include/llvm/SandboxIR/Utils.h

```diff
@@ -60,6 +60,16 @@ class Utils {
         getUnderlyingObject(LSI->getPointerOperand()->Val));
   }
 
+  /// \Returns the number of elements in \p Ty, that is the number of lanes
```

Comment:
Right now there is no VecUtils.h in the sandbox vectorizer. Let's wait to do this until there are enough functions where it makes sense.

Reply:
I am actually using this exact function in one of the Legality patches and I placed it in `VecUtils.h`, but it's up to you if you want to create the file now or later, either way is fine with me. If you decide to keep it in `Utils.h` please add a TODO so that we remember to clean it up later.